test/unit/constructor.js

```diff
@@ -2,16 +2,16 @@ import { define } from '../../src/index';
 import helperElement from '../lib/element';
 import resolved from '../lib/resolved';
 
-describe('constructor', function () {
-  var id;
+describe('constructor', () => {
```

Comment:
not sure if we should be using arrow functions with mocha based on https://mochajs.org/#arrow-functions ?